mutablesecurity/solutions/implementations/duplicati/code.py

```diff
@@ -0,0 +1,693 @@
+    INFO_TYPE = StringDataType
+    PROPERTIES = [
+        InformationProperties.CONFIGURATION,
+        InformationProperties.OPTIONAL,
+        InformationProperties.WITH_DEFAULT_VALUE,
+        InformationProperties.NON_DEDUCTIBLE,
+        InformationProperties.WRITABLE,
+    ]
+    DEFAULT_VALUE = "zip"
+    GETTER = CompressionModuleValue
+    SETTER = set_configuration
+
+
+class SkipFilesLarger(BaseInformation):
+    @staticmethod
+    @deploy
+    def set_configuration(
+        old_value: typing.Any, new_value: typing.Any
+    ) -> None:
+        template_path = os.path.join(
+            os.path.dirname(__file__), "files/duplicati.conf.j2"
+        )
+        j2_values = {"skip_files_larger_than": SkipFilesLarger.get()}
+        files.template(
+            src=template_path,
+            dest="/opt/mutablesecurity/duplicati/duplicati.conf",
+            configuration=j2_values,
+            name="Copy the generated configuration into duplicati's folder.",
+        )
+
+    @staticmethod
+    @deploy
+    class SkipLargerFilesValue(FactBase):
+        command = "echo 'hi'"
+
+        @staticmethod
+        def process(output: typing.List[str]) -> str:
+            return (
+                "cat /opt/mutablesecurity/duplicati/duplicati.conf | grep"
+                f" {IDENTIFIER} | cut -d : -f 2"
+            )
+
+    IDENTIFIER = "skip_files_larger_than"
+    DESCRIPTION = (
+        "Don't backup files which heve size larger than corresponding value"
+    )
+    INFO_TYPE = StringDataType
+    PROPERTIES = [
+        InformationProperties.CONFIGURATION,
+        InformationProperties.OPTIONAL,
+        InformationProperties.WITH_DEFAULT_VALUE,
+        InformationProperties.NON_DEDUCTIBLE,
+        InformationProperties.WRITABLE,
+    ]
+    DEFAULT_VALUE = "2GB"
+    GETTER = SkipLargerFilesValue
+    SETTER = set_configuration
+
+
+class ExcludeFilesAttributes(BaseInformation):
+    @staticmethod
+    @deploy
+    def set_configuration(
+        old_value: typing.Any, new_value: typing.Any
+    ) -> None:
+        template_path = os.path.join(
+            os.path.dirname(__file__), "files/duplicati.conf.j2"
+        )
+        j2_values = {"exclude_files_attributes": ExcludeFilesAttributes.get()}
+        files.template(
+            src=template_path,
+            dest="/opt/mutablesecurity/duplicati/duplicati.conf",
+            configuration=j2_values,
+            name="Copy the generated configuration into duplicati's folder.",
+        )
+
+    @staticmethod
+    @deploy
+    class ExcludeFilesValue(FactBase):
+        command = "echo 'hi'"
+
+        @staticmethod
+        def process(output: typing.List[str]) -> str:
+            return (
+                "cat /opt/mutablesecurity/duplicati/duplicati.conf | grep"
+                f" {IDENTIFIER} | cut -d : -f 2"
+            )
+
+    IDENTIFIER = "exclude_files_attributes"
+    DESCRIPTION = "Don't backup files which have this attribute"
+    INFO_TYPE = StringDataType
```

Comment:
From [this comment](https://forum.duplicati.com/t/2-0-3-5-ignoring-exclude-files-attributes/3755/2) on Duplicati's forum, I understand that there are multiple attributes that can be excluded. Please transform this into an array of string, such as users may skip multiple file types.